terraform/modules/task-definitions/content-store/main.tf

```diff
@@ -49,7 +49,7 @@ module "task_definition" {
       "image" : "govuk/content-store:${var.image_tag}", # TODO: replace temporary label
       "essential" : true,
       "environment" : [
-        { "name" : "APPMESH_VIRTUAL_NODE_NAME", "value" : "mesh/${var.mesh_name}/virtualNode/${var.service_name}" },
+        { "name" : "APPMESH_RESOURCE_ARN", "value" : "mesh/${var.mesh_name}/virtualNode/${var.service_name}" },
```

Comment:
Does this actually need to be on the app containers in the first place? I'm fairly sure it's only used by Envoy.

Reply:
I replaced it everywhere based on another review, I can delete them if not needed.